mitodl/open-discussions
# Remove writing bootcamps to Course model

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #2086 

#### What's this PR do?
Removes the code that is populating bootcamps into the `Course` table and depopulates such records from the table

#### How should this be manually tested?
The migration should run and you should be able to run the task `course_catalog.tasks.get_bootcamp_data` without seeing an error or seeing bootcamp records populate into `Course`
